.gitignore

```diff
@@ -0,0 +1 @@
+artifacts/
```

Comment:
should this be `/artifacts/` (the directory in the root) or `artifacts/` (any directory in the project)?

Reply:
Ack, I thought I had put the `/` prefix, will fix!